examples/gno.land/p/demo/int256/bitwise.gno

```diff
@@ -0,0 +1,93 @@
+package int256
```

Comment:
Please add unit tests for methods / functions in this file 🙏

Reply:
516c99575b238652847e4ceb6c9b40ad2b0cc4e5